aws-env.dev

```diff
@@ -1,22 +1,44 @@
-getparam () {
-	aws --profile ins-dev ssm get-parameters --names "$1" --with-decryption --query Parameters[0].Value --output text
-}
-export DOMAIN=ins.unee-t.com
+#!/bin/bash
+### This file stores all environment variables for the DEV environment for this repo
+
+# This script is called as part of the Travis CI deployment routine
+# We can use the default aws profile configured in Travis CI settings to get the variables we need
+
+# We create a procedure to retrieve the variables from the AWS parameter store
+# Make sure to 
+#   - configure your AWS CLI accordingly
+#   - update the profile you need to use to access these variables if needed
+#	- PROFILE is set when .travis.yml run 
+
+	getparam () {
+		aws --profile ${PROFILE} ssm get-parameters --names "$1" --with-decryption --query Parameters[0].Value --output text
+	}
+
+# We prepare the hardcoded variables that we will need to deploy what's needed
+
 export STAGE=dev
```

Comment:
you should use 
` export STAGE=$(getparam STAGE)` 
instead of a hard coded variable

Reply:
Fixed in https://github.com/Unee-T-INS/frontend/pull/7/commits/9cf1fad916eb0293046067cb7016b7781795933e